### ISSUE TITLE
add option on media playing

### ISSUE THREAD

**Issue by waonemail1979** [solid-software/flutter_vlc_player#88]
hi,
in example vlc plugin can set options on init. is it possible to add / change vlc options on media playing when user needed.
example add / change audio channel to '--stereo-mode=3' etc.

sorry for my bad english and thanks for your time.

**Comment by mitchross** [comment]
yes see https://github.com/solid-software/flutter_vlc_player/pull/84

**Comment by waonemail1979** [comment]
hi
i try to add parameter option: '--stereo-mode=' on media running / playing but not working, is there a clue for me ?

**Comment by waonemail1979** [comment]
return Scaffold(
      body: new VlcPlayer(
        aspectRatio: 16 / 9,
        url: "http://188.166.250.67/smartkaraoke/a/song_00000000_000000.mpg",
        controller: _videoViewController,
        // Play with vlc options
        options: getOptions(),
        hwAcc: HwAcc.DISABLED, // or {HwAcc.DISABLED, HwAcc.AUTO, HwAcc.DECODING, HwAcc.FULL}
        placeholder: new Center(
            child: new CircularProgressIndicator()
        ),
      ),
    );

  List<String> getOptions(){
    if (nstereo == 3) {
      return [
        '--quiet',
        '--no-drop-late-frames',
        '--no-skip-frames',
        '--rtsp-tcp',
        '--rtp-timeout=15',
        '--network-caching=5000',
        '--stereo-mode=3'
      ];
    } else if (nstereo == 4){
      return [
        '--quiet',
        '--no-drop-late-frames',
        '--no-skip-frames',
        '--rtsp-tcp',
        '--rtp-timeout=15',
        '--network-caching=5000',
        '--stereo-mode=4'
      ];
    } else {
      return [
        '--quiet',
        '--no-drop-late-frames',
        '--no-skip-frames',
        '--rtsp-tcp',
        '--rtp-timeout=15',
        '--network-caching=5000',
        '--stereo-mode=5'
      ];
    }
  }

**Comment by waonemail1979** [comment]
return Scaffold(
      body: new VlcPlayer(
        aspectRatio: 16 / 9,
        url: "http://server_name/file.mpg",
        controller: _videoViewController,
        // Play with vlc options
        options: getOptions(),
        hwAcc: HwAcc.DISABLED, // or {HwAcc.DISABLED, HwAcc.AUTO, HwAcc.DECODING, HwAcc.FULL}
        placeholder: new Center(
            child: new CircularProgressIndicator()
        ),
      ),
    );

  List<String> getOptions(){
    if (nstereo == 3) {
      return [
        '--quiet',
        '--no-drop-late-frames',
        '--no-skip-frames',
        '--rtsp-tcp',
        '--rtp-timeout=15',
        '--network-caching=5000',
        '--stereo-mode=3'
      ];
    } else if (nstereo == 4){
      return [
        '--quiet',
        '--no-drop-late-frames',
        '--no-skip-frames',
        '--rtsp-tcp',
        '--rtp-timeout=15',
        '--network-caching=5000',
        '--stereo-mode=4'
      ];
    } else {
      return [
        '--quiet',
        '--no-drop-late-frames',
        '--no-skip-frames',
        '--rtsp-tcp',
        '--rtp-timeout=15',
        '--network-caching=5000',
        '--stereo-mode=5'
      ];
    }
  }

**Comment by mitchross** [comment]
Can you try to get it working with regular VLC command line? https://wiki.videolan.org/Documentation:Streaming_HowTo/Command_Line_Examples/

If you can, let me know
